master/tcs.yaml

```diff
@@ -298,16 +298,15 @@
     'pro parte synonym' or 'misapplication'.
   usage: null
   notes: >-
-    This is an IRI term. One should use a value from the TDWG Taxon Concept 
-    Relationship Type Vocabulary. In the case of Taxon Concept Relationships 
-    from traditional synonymy, the `relationshipAccordingTo` is the same as the 
-    `accordingTo` of the Taxon Concept that is the `subjectTaxonConcept`.
+    This term should be used together with `relationshipType` (which is required 
```

Comment:
... so I can't remember the discussions about this that clearly. The relevant GH discussion seems to be in #65, in particular a potential disagreement (between @Archilegt and you) about whether synonyms should be properties of names or TCs. I agree with you that to make a synonymy statement one must have a TC to work with, and in general the most accurate way to encode synonymy is using `TC intersects TC`. But... in practice most synonymy statements have no "sec." for the names so they appear to be properties of names. Have we discussed writing any general guidance for users which would allow/encourage `TC name X; accordingTo NULL`? I.e., forcing a name to be a concept, but with unknown TaxonTreatment. If we will allow this, then it makes sense to model synonym as a TC property. If this is not allowed, we must make synonymy a property of names, otherwise vast amounts of taxonomy information cannot be correctly encoded.